### PR TITLE
golang/gorm example implementation

### PR DESCRIPTION
**Pull request by azeemshahzad**

### CODE REVIEW

File: golang/gorm/src/controller/orderController.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	}
+	
+	w.Header().Add("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(orders)
+}
+
+/* handler function for GET method */
+var SelectOrder = func (w http.ResponseWriter, r *http.Request) {
+	orderId := mux.Vars(r)["orderId"]
+	fmt.Println(fmt.Sprintf("finding order with order_id = %s", orderId))
```
Comment (rajukumaryb) [inline golang/gorm/src/controller/orderController.go]: Why is fmt.Printf() not preferable to fmt.Println(fmt.Sprintf()) ? Is this in anticipation of using the intermediate string for some other purpose as well ?

File: golang/gorm/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Compile, Deploy and Run
+Install required components
+```
+go get github.com/google/uuid
+go get github.com/gorilla/mux
```
Comment (rajukumaryb) [inline golang/gorm/README.md]: "github.com/jinzhu/gorm" seems to be a requirement as well

**Comment by rajukumaryb** [comment]
"cd orm-examples/golang/gorm/src/model; go vet" finds the following issues. Are these to be ignored ?

# model
./orderModel.go:10:2: struct field tag `gorm:"association_foreignkey:OrderId` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
./orderModel.go:12:2: struct field tag `gorm:"association_foreignkey:ProductId` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
./orderModel.go:22:2: struct field tag `gorm:"primary_key";gorm:"AUTO_INCREMENT"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces
./orderModel.go:25:2: struct field tag `gorm:"association_foreignkey:UserId` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
./productModel.go:8:2: struct field tag `gorm:"primary_key";gorm:"AUTO_INCREMENT"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces
./userModel.go:6:2: struct field tag `gorm:"primary_key";gorm:"AUTO_INCREMENT"` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces

File: golang/gorm/README.md (hunk 2 above)
Comment (azeemshahzad) [reply golang/gorm/README.md]: think gorm is basic requirement just like golang itself, rather than a dependency. but added that anyway.

**Comment by azeemshahzad** [comment]
Though there wasn't any problem with the functionality, but I made the recommended changes..

File: golang/gorm/src/config/config.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+	"platform" : "postgres",
+	"host" : "localhost",
+	"dbport" : 5432,
```
Comment (m-iancu) [inline golang/gorm/src/config/config.json]: Should be `5433`.

**Pull request merged**